soc/arm/nxp_lpc/lpc55xxx/soc.c

```diff
@@ -67,50 +82,93 @@ static ALWAYS_INLINE void clock_init(void)
 #if defined(CONFIG_SOC_LPC55S06) || defined(CONFIG_SOC_LPC55S16) || \
 	defined(CONFIG_SOC_LPC55S28) || defined(CONFIG_SOC_LPC55S36) || \
 	defined(CONFIG_SOC_LPC55S69_CPU0)
-    /*!< Set up the clock sources */
-    /*!< Configure FRO192M */
-	/*!< Ensure FRO is on  */
+	/* Set up the clock sources */
+	/* Configure FRO192M */
+	/* Ensure FRO is on  */
 	POWER_DisablePD(kPDRUNCFG_PD_FRO192M);
-	/*!< Set up FRO to the 12 MHz, just for sure */
+	/* Set up FRO to the 12 MHz, to ensure we can change the clock freq */
```

Comment:
duplicate comment as line 91

Reply:
The first line indicating the function below it helps us see that we can adjust the frequency of the clock, the second function shows that we can actually adjust the selection of the clock that the said frequency will go to. The ending parts of these comments explain this, perhaps not well enough, if you think this should be changed please let me know.

Reply:
well, it doesnt really matter, I just didn't know if it was on purpose